pkg/controller/declare.go

```diff
@@ -47,7 +47,8 @@ func (c *controller) HandleDeclare(request *brevity.DeclareRequest) {
 			logger.Warn().Stringer("bearing", request.Bearing).Msg("bearing provided to HandleDeclare should be magnetic")
 		}
 		origin = trackfile.LastKnown().Point
-		bearing = request.Bearing
+		declination := c.scope.Declination(origin)
+		bearing = request.Bearing.True(declination)
```

Comment:
Bugfix here - we were passing a magnetic bearing to orb/geo which would have given us a slightly wrong BRAA